open_instruct/vllm_utils3.py

```diff
@@ -566,6 +599,37 @@ def wake_up(self, tags: Optional[list[str]] = None):
     def ready(self):
         return True
 
+    def get_kv_cache_info(self):
```

Comment:
this is super cool, but how flexible do you think this is wrt future vllm versions? will this break if I set vllm to the v0 engine?

Reply:
I think it's fine? They're not private variables, so I think it's safe. 

I'm not sure about v0 vs v1. I am not testing against v0. I think we should pick one and test everything against that version. If you want to move to v0 I am open to that.